web-app/src/store/user/getters.ts

```diff
@@ -0,0 +1,20 @@
+import { GetterTree } from 'vuex';
+import { StateInterface } from '../index';
+import { UserStateInterface } from './state';
+
+const getters: GetterTree<UserStateInterface, StateInterface> = {
+  id(state: UserStateInterface) {
+    return state.id;
+  },
+  tenantId(state: UserStateInterface) {
+    return state.tenantId;
+  },
+  name(state: UserStateInterface) {
+    return state.name;
+  },
+  username(state: UserStateInterface) {
+    return state.username;
+  },
```

Comment:
getters are for custom data. e.q. `fullName()`
raw properties are available from state e.q. `this.$store.state.currentUser.id`

Reply:
Getting raw properties is not a good practice. We should avoid it.